ckan/.snyk

```diff
@@ -80,4 +80,22 @@ ignore:
            No remediation available yet. Issue tracked in github:
            https://github.com/GSA/data.gov/issues/4782
         expires: 2024-09-30T16:20:58.017Z
+  SNYK-PYTHON-CKAN-7786366:
```

Comment:
where did you find these numbers?

Reply:
They are in the latest snyk scan report
https://github.com/GSA/catalog.data.gov/actions/runs/10559448879/job/29250897911#step:6:36

![image](https://github.com/user-attachments/assets/fe88c72a-cfdd-4664-8036-55ab6aab6438)

Reply:
Thanks!